TLSharp.Core/TelegramClient.cs

```diff
@@ -110,15 +112,15 @@ public bool IsUserAuthorized()
 
             var authCheckPhoneRequest = new TLRequestCheckPhone() { phone_number = phoneNumber };
             var completed = false;
-            while(!completed)
+            while (!completed)
```

Comment:
Since we got here, a few things needs be cleared up:
- First, I'm contributing because I have made some changes to the library that I have used in my project and I thought everyone could benefit from it. This doesn't mean that I'm dedicated my time to work on that library anymore. Also it means that my focus is on the features rather than consistency.
- Second, reformatting/resetting/reverting/etc takes more time for me than you as the owner, because I don't have the environment and the tools/settings as you do. I as a contributor should try to match the original guidelines for sure, but this has limits.
- Third, being able to propose a clean PRs depends on the importance of the project for me of course. I always respect projects and the time that have put on and try to follow what rules they have but this comes last. FYI I have made changes  to my VS to match your settings and still these changes you mentioned got out of hand. In the end this was the best i could do in the time I had for it.
- Forth and last, It is just a PR and you got the idea, if you don't like the code, reject it and implement it yourself. It doesn't really matter.